### PR TITLE
Create Dylan-Scruggs-Brake-Subsystem-Signoff.md

### PR DESCRIPTION
**Pull request by Dylan2432**
Requesting signoff for Brake Subsystem

### CODE REVIEW

File: Documentation/Signoffs/Dylan-Scruggs-Brake-Subsystem-Signoff.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+## Functionality
+ &nbsp;&nbsp;&nbsp;&nbsp;&nbsp; The Brake Control System is being implemented to allow for simulation of the bike's brake to be captured through controlled inputs. This system will be the driving factor behind ensuring the customer's desires are satisfied. With this system the user shall be able to apply a desired load to observe how the bike reacts to different braking and road grade loads. This will be made possible with the implementation of a 3-phase AC electric motor, connected to the drum mechanically, to operate as a generator. With this, a load bank will be attached to the output of the generator to dissipate generated electrical energy in order to provide different loads. Through the coaction of these components, this subsystem will be able to deliver desired metrics over various simulated conditions.
+
+## Constraints
+|No.        |Constraints   |Origin   |
+|---|---------------------|--------|
+|2|Be able to safely operate the motorcycle up to 74 km/h.         |Safety Constraint |
+|3|Capture current and voltage supplied to the drive motor along with vehicle speed, torque, horsepower, and rpm and display all results.       |Design Constraint|
```
Comment (JesseTNRoberts) [inline Documentation/Signoffs/Dylan-Scruggs-Brake-Subsystem-Signoff.md]: What if the current is 1000amps and the voltage is 10^10 volts?

File: Documentation/Signoffs/Dylan-Scruggs-Brake-Subsystem-Signoff.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+|5|Comply with all relevant codes and standards listed in II. Ethical, Professional, and Standard Considerations, Section B below.         |Safety Constraint |
```
Comment (JesseTNRoberts) [inline Documentation/Signoffs/Dylan-Scruggs-Brake-Subsystem-Signoff.md]: ambiguous.

File: Documentation/Signoffs/Dylan-Scruggs-Brake-Subsystem-Signoff.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+|7|Support incline testing up to 8% road grade.       |Design Constraint|
```
Comment (JesseTNRoberts) [inline Documentation/Signoffs/Dylan-Scruggs-Brake-Subsystem-Signoff.md]: How can you ensure that?

File: Documentation/Signoffs/Dylan-Scruggs-Brake-Subsystem-Signoff.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+|8|Support repeatedly testing the bike from 0km/h to 50km/h and then back down to 0km/h.         |Design Constraint |
```
Comment (JesseTNRoberts) [inline Documentation/Signoffs/Dylan-Scruggs-Brake-Subsystem-Signoff.md]: What does support here mean?

File: Documentation/Signoffs/Dylan-Scruggs-Brake-Subsystem-Signoff.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+|13|Be able to simulate the braking system of the motorcycle. (50km/h to 30km/h)       |Design Constraint|
```
Comment (JesseTNRoberts) [inline Documentation/Signoffs/Dylan-Scruggs-Brake-Subsystem-Signoff.md]: What does this mean?

**Review by JesseTNRoberts — COMMENTED** [review]
What does safely mean?

**Review by JesseTNRoberts — COMMENTED** [review]
I don't understand how this is a brake subsystem? It seems to be the load bank?

**Comment by micahr234** [comment]
Please remove Jesse and add me... when you are ready for review.

**Pull request closed without merging**